indigo/distribution.yaml

```diff
@@ -232,6 +232,22 @@ repositories:
       url: https://github.com/rosalfred/alfred_sr_linux.git
       version: master
     status: developed
+  alliance:
+    release:
+      packages:
+      - alliance
+      - alliance_msgs
+      - rqt_alliance
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/adrianohrl/alliance-release.git
+      version: 1.0.5-0
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/adrianohrl/alliance.git
+      version: '1.05'
```

Comment:
This needs to point to a valid branch name instead.

Reply:
It should be, at line 249:

version: '1.0.5'

Reply:
That is a tag but it needs to be a branch name.

Reply:
master or indigo-devel

Reply:
Whichever contains the code you want to be used for CI testing.